dispatches/models/fossil_case/thermal_oil/thermal_oil.py

```diff
@@ -100,7 +100,7 @@ class _StateBlock(StateBlock):
     def initialize(self, state_args={}, state_vars_fixed=False,
                    hold_state=False, outlvl=idaeslog.NOTSET,
                    temperature_bounds=(260, 616),
-                   solver='ipopt', optarg={'tol': 1e-8}):
+                   solver=None, optarg={'tol': 1e-8}):
```

Comment:
Default for `optarg` should be `{}`.

Reply:
In case anyone thinks of #330, that needs `get_solver` to be fixed first.

Reply:
Isn't default `optarg=None` ? In either case this needs updating